### PR TITLE
Issue 190

### LINKED ISSUE #190: support "source filename.sql"  or ". filename"

### PR DESCRIPTION
**Pull request by phdru**
Fix #190.

### CODE REVIEW

File: sqlparse/lexer.py [hunk 1]
```diff
@@ -20,6 +20,13 @@
 from sqlparse.utils import consume
 
 
+try:
+    file
```
Comment (vmuriart) [inline sqlparse/lexer.py]: whats the purposed of this line ?

**Comment by vmuriart** [comment]
Can you add a couple tests here as well

**Comment by vmuriart** [comment]
Wouldn't the next line do the same thing? It should still raise the same Exception (never tried it though)

**Comment by phdru** [comment]
Oh, yes. :smile:

**Comment by phdru** [comment]
Done. Fixed and added a test. All tests passed.

**Comment by vmuriart** [comment]
Thanks for the pr!

**Pull request merged**